cmd/ocm/list/cmd.go

```diff
@@ -46,4 +47,5 @@ func init() {
 	Cmd.AddCommand(upgradepolicy.Cmd)
 	Cmd.AddCommand(user.Cmd)
 	Cmd.AddCommand(version.Cmd)
+	Cmd.AddCommand(rhRegion.Cmd)
```

Comment:
is there a way to make the command hidden like the `--rh-region` flag?

Reply:
Hid the command